cypress/e2e/course_board/createEditDeleteSingleColumnBoard.feature

```diff
@@ -0,0 +1,112 @@
+        # Note: this step is not applicable for the admin user
+        #Then I see the course 'CypressAut Test Creation and Deletion' on the course overview page
+
+        # teacher adds a new Board
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        When I go to courses overview
+        When I go to course '<course_name>'
+        When I go to the tab contents in course detail page
+        When I click on FAB to create new content
+        When I click on the button FAB New Column Board
+        Then I see a dialog box for column board
+        Then I see in dialog box option for multi-column board
+        Then I see in dialog box option for single column board
+        When I choose single-column board in the dialog box
+        Then I see the page Course Board details
+        When I click on the button three dot menu in course board
+        When I click on the option Edit in three dot menu in course board
+        Then I enter the course board title '<board_title>'
+        When I click on the page outside of the column
+        # Then I see the course Board name '<board_title>'
+        Then I see the chip Draft in the course board
+
+        # student does not see the board
+        Given I am logged in as a '<student>' at '<namespace>'
+        When I go to courses overview
+        When I go to course '<course_name>'
+        Then I can not see content '<board_title>'
+
+        # teacher publishs the column board
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        When I go to courses overview
+        When I go to course '<course_name>'
+        When I click on card Course Board
+        When I click on the button three dot menu in course board
+        When I click on the option Publish in three dot menu in course board
+        Then I do not see the chip Draft in the course board
+
+        # teacher adds a new column with whiteboard in the Board
+        When I click on the button Add column in the course board
+        When I enter the title name '<column_board_title>' in the column
+        When I click on the page outside of the column
+        Then I see my column named '<column_board_title>'
+        Then I click on the button with the Icon Plus to add a new card in the column
+        When I enter '<card_text_content>' to board card text element
+        When I click on plus icon to add content into card
+        When I select whiteboard from the menu
+        Then I see a whiteboard on the board
+        Then I see '<card_text_content>' in board card text element
+
+        # student sees the board
+        Given I am logged in as a '<student>' at '<namespace>'
+        When I go to courses overview
+        When I go to course '<course_name>'
+        Then I can see board '<board_title>' on course page
+        When I click on card Course Board
+        Then I see a whiteboard on the board
+        Then I see '<card_text_content>' in board card text element
+
```

Comment:
steps for editing of the column board is missing in the feature file. Is it intended or not, as in the feature file it was mention that we are doing CRUD for column board.